auditlog/models/rule.py

```diff
@@ -237,44 +237,26 @@ def _make_read(self):
         self.ensure_one()
         log_type = self.log_type
 
-        def read(self, *args, **kwargs):
-            result = read.origin(self, *args, **kwargs)
-            # Sometimes the result is not a list but a dictionary
-            # Also, we can not modify the current result as it will break calls
+        @api.multi
+        def read(self, fields=None, load='_classic_read'):
+            result = read.origin(self, fields, load)
             result2 = result
             if not isinstance(result2, list):
                 result2 = [result]
             read_values = dict((d['id'], d) for d in result2)
```

Comment:
`result`/`result2` trick isn't necessary anymore as the `read` method is only compatible with the new API. Check the piece of code I sent in the other comment.